app/views/tasks/_form.html.haml

```diff
@@ -9,7 +9,7 @@
   .field
     =f.label :name
     =f.text_field :name
-  .field
+  .field{:class => ("hidden" if @task.new_record?)}
```

Comment:
If a pesky user wanted to edit this hidden field, is there another way to ensure new tasks aren't marked as completed?